src/toolbar/toolbar.jsx

```diff
@@ -19,6 +19,8 @@ function getStyles(props, state) {
       height: toolbar.height,
       width: '100%',
       padding: noGutter ? 0 : `0px ${baseTheme.spacing.desktopGutter}px`,
+      display: 'inline-flex',
```

Comment:
@nathanmarks : I'm sorry about the inline-flex thing. It was something I was trying but forgot to change it back to flex. Flex does not break any changes and it works fine. I changed it back.